.github/workflows/rav1e.yml

```diff
@@ -268,9 +268,9 @@ jobs:
         CARGO_INCREMENTAL: 0
         RUSTFLAGS: >
           -Zprofile -Ccodegen-units=1 -Clink-dead-code -Coverflow-checks=off
-          -Zno-landing-pads
+          -Cpanic=abort -Zpanic_abort_tests
```

Comment:
Is this equivalent to -Zno-landing-pads?

Reply:
Yes

Reply:
Yes

Reply:
Add also `RUSTDOCFLAGS: -Cpanic=abort` to avoid problems with doc-tests

Reply:
Sure :)